spec/catalog.yaml

```diff
@@ -150,6 +169,18 @@ components:
           example: { "created_at": "1452120468" }
           default: { }
 
+    ListNamespacesResponse:
+      type: object
+      properties:
```

Comment:
I remember you had parameters like `start_after`, do you plan to add that in later PRs?

Reply:
Yes, currently, I have removed it to let the API minimize availability. Let's enrich it soon. I will file a ticket to track this feature. WDTY?